tests/std/tests/parallel_algorithms.lst

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier Apache-2.0 WITH LLVM-exception
+
+RUNALL_INCLUDE .\native_17_matrix.lst

```

Comment:
Did you intend to do something here other than adding a layer of indirection? Cross with `/O2`, perhaps?

Reply:
Oh, yes!